qiskit/pulse/commands/sample_pulse.py

```diff
@@ -104,27 +105,34 @@ def draw(self, dt: float = 1,
              style: Optional['PulseStyle'] = None,
              filename: Optional[str] = None,
              interp_method: Optional[Callable] = None,
-             scaling: float = 1, interactive: bool = False):
+             scale: float = 1, interactive: bool = False,
+             scaling: float = 1):
         """Plot the interpolated envelope of pulse.
 
         Args:
             dt: Time interval of samples.
             style: A style sheet to configure plot appearance
             filename: Name required to save pulse image
             interp_method: A function for interpolation
-            scaling: Relative visual scaling of waveform amplitudes
+            scale: Relative visual scaling of waveform amplitudes
             interactive: When set true show the circuit in a new window
                 (this depends on the matplotlib backend being used supporting this)
+            scaling: Deprecated, see `scale`
 
         Returns:
             matplotlib.figure: A matplotlib figure object of the pulse envelope
         """
         # pylint: disable=invalid-name, cyclic-import
+        if scaling:
```

Comment:
```suggestion
        if scaling is not None:
```
Let me know if you have a reason you prefer not to do this:)